### ISSUE TITLE
Function Naming

### ISSUE THREAD

**Issue by beyzasgnms** [Trendyol/ios-guidelines#71]
Function Naming Guidelines
	•	Avoid using `get` in function names: Instead of naming functions like `getUser(),` prefer more descriptive names that convey the action being performed. For example, use `fetchUser()` or `retrieveUser()` to provide clearer intent and avoid ambiguity. This practice helps in making function names more descriptive and aligned with their actual purpose.

**Comment by eyupgoymen** [comment]
I agree, since "get" has several meanings, we should be selective for prefixes. We can also consider properties and computed properties besides functions, they should not have "get" prefix too imo.

**Comment by aniltaskiran** [comment]
Earlier version of this document has that logic. Actaully we choose it with team. I think existing rules should be stay. For example this one. 

![CleanShot 2024-08-02 at 12 08 54@2x](https://github.com/user-attachments/assets/f829b28b-c418-4d7a-b4f4-33df920f4121)

https://github.com/Trendyol/ios-guidelines/blob/b635320c45f40b8cd76ed4d0b18fc37a2ab84fe5/code_style_guideline/code_style_guideline.md#naming

**Comment by aniltaskiran** [comment]
By the way, if you removed everything from the previous rules, we will talk about these topics again. I think this will make us waste time. Like this issue, we already agreed on that :D

**Comment by eyupgoymen** [comment]
Since this rule was part of the previous guide and not existing in our guide, Workgroup will re-review existing rules by considering your comments next week and let you guys know about the action, thanks for your feedbacks 🙌🏻
